Formula/emoji-cheat-sheet-plus.rb

```diff
@@ -7,24 +7,14 @@ class EmojiCheatSheetPlus < EmacsFormula
   sha256 "7f8592a2a54e8fd58689f8f6a6d118b9511e48ce56fa84f294fa6623333c7d5a"
   head "https://github.com/syl20bnr/emacs-emoji-cheat-sheet-plus.git"
 
-  # due to helm
-  depends_on :emacs => "24.3"
+  depends_on :emacs => ["24.3", "with-imagemagick"]
```

Comment:
FYI, `with-imagemagick` won't take affect if emacs is already installed. It will only work for a fresh installation.